web/basic/login.html

```diff
@@ -6,9 +6,25 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
-    <meta name="description" content="txAdmin - remotely Manage & Monitor your GTA5 FiveM Server">
+    <title>txAdmin | {{it.headerTitle}}</title>
```

Comment:
This should be `{{it.headerTitle}} | txAdmin`

Reply:
guess so

Reply:
Actually no, header title logic is in the rendered function